README.md

```diff
@@ -274,7 +274,8 @@ Error code, which comes from the module side.
 |`APP_UNAUTHORIZED`||
 |`PROTOCOL_VIOLATION`||
 |`UNSUPPORTED_HMI_RESOURCE`||
-|`RESOURCE_CONSTRAINT`||
+|`RESOURCE_CONSTRAINT`|By sending this value, SDL unregisters the application to allow the HMI to close the application.|
```

Comment:
@LitvinenkoIra  I believe this is not related to proposal

Reply:
@AKalinich-Luxoft Agree, removed. Please check the latest updates